src/fastoad/cmd/fast.py

```diff
@@ -266,6 +266,7 @@ def run(self):
             help="Provides the identifiers of available systems",
             description="Provides the identifiers of available systems",
         )
+        self._add_conf_file_argument(parser_list_modules, required=False)
```

Comment:
This line has not to be put back, since the argument "source_path" is now defined below.
```suggestion
```

Reply:
Thanks. Corrected in 69af999.
I prefer solution 1 as `om.list_modules()` is working as expected.